.github/CODEOWNERS

```diff
@@ -27,19 +27,24 @@ go.sum                    @DataDog/corpweb @DataDog/documentation
 # Agent Shared Component
 /content/en/agent/proxy.md                              @DataDog/agent-shared-components @DataDog/documentation
 
-# Partners
-/content/en/partners/                                   @DataDog/partner-tech-solutions @DataDog/documentation
+# Agent-integration-tools
+content/en/developers/integrations/agent_integration.md                         @DataDog/agent-integrations @DataDog/documentation
+content/en/developers/integrations/python.md                                    @DataDog/agent-integrations @DataDog/documentation
+content/en/developers/integrations/guide/legacy.md                              @DataDog/agent-integrations @DataDog/documentation
 
 # Marketplace
-
-/content/en/developers/integrations/agent_integration/                     @Datadog/marketplace-product-management @DataDog/documentation
-/content/en/developers/integrations/api_integration/                       @Datadog/marketplace-product-management @DataDog/documentation
-/content/en/developers/integrations/marketplace_offering/                  @Datadog/marketplace-product-management @DataDog/documentation
-/content/en/developers/integrations/check_references/                      @Datadog/marketplace-product-management @DataDog/documentation
-/content/en/developers/integrations/create_a_tile/                         @Datadog/marketplace-product-management @DataDog/documentation
-/layouts/shortcodes/integration-assets.md                                  @Datadog/marketplace-product-management @DataDog/documentation
-/layouts/shortcodes/integration-assets-reference.md                        @Datadog/marketplace-product-management @DataDog/documentation
-/layouts/shortcodes/integration_categories.md                              @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/api_integration.md                           @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/log_integration.md                           @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/marketplace_offering.md                      @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/create_a_tile.md                             @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/oauth_for_integrations.md                    @Datadog/marketplace-product-management @DataDog/documentation
+content/en/developers/integrations/create_an_integration_dashboard.md           @Datadog/marketplace-product-management @DataDog/documentation
```

Comment:
Agent pms are the owners for dashboards and any updates to them. I think they should be code owners here!